2-projects/1-todo-list/index.html

```diff
@@ -26,26 +26,7 @@
           <div class="col-sm">
             <ul id="todo-list" class="list-group">
               <!-- This is how the todos look in html - try to make the functionality to create them using Javascript - don't worry about the class names for now just copy and paste them -->
-              <li
-                class="list-group-item d-flex justify-content-between align-items-center"
-              >
-                Wash the dishes
```

Comment:
You should remove these lists of Todos because you are using javascript to add and display does as the task suggested in the readMe file.